docs/whats-new/dotnet-aspire-9.2.md

```diff
@@ -0,0 +1,108 @@
+
+- [.NET support policy](https://dotnet.microsoft.com/platform/support/policy): Definitions for LTS and STS.
+- [.NET Aspire support policy](https://dotnet.microsoft.com/platform/support/policy/aspire): Important unique product life cycle details.
+
+## ⬆️ Upgrade to .NET Aspire 9.2
+
+Moving between minor releases of .NET Aspire is simple:
+
+1. In your app host project file (that is, _MyApp.AppHost.csproj_), update the [📦 Aspire.AppHost.Sdk](https://www.nuget.org/packages/Aspire.AppHost.Sdk) NuGet package to version `9.2.0`:
+
+    ```diff
+    <Project Sdk="Microsoft.NET.Sdk">
+
+        <Sdk Name="Aspire.AppHost.Sdk" Version="9.2.0" />
+        
+        <PropertyGroup>
+            <OutputType>Exe</OutputType>
+            <TargetFramework>net9.0</TargetFramework>
+    -       <IsAspireHost>true</IsAspireHost>
+            <!-- Omitted for brevity -->
+        </PropertyGroup>
+        
+        <ItemGroup>
+            <PackageReference Include="Aspire.Hosting.AppHost" Version="9.2.0" />
+        </ItemGroup>
+    
+        <!-- Omitted for brevity -->
+    </Project>
+    ```
+
+    > [!IMPORTANT]
+    > The `IsAspireHost` property is no longer required in the project file. For more information, see [App host project file changes](#-app-host-project-file-changes).
+
+    For more information, see [.NET Aspire SDK](xref:dotnet/aspire/sdk).
+
+1. Check for any NuGet package updates, either using the NuGet Package Manager in Visual Studio or the **Update NuGet Package** command in VS Code.
+1. Update to the latest [.NET Aspire templates](../fundamentals/aspire-sdk-templates.md) by running the following .NET command line:
+
+    ```dotnetcli
+    dotnet new update
+    ```
+
+    > [!IMPORTANT]
+    > The `dotnet new update` command updates all of your templates to the latest version.
+
+If your app host project file doesn't have the `Aspire.AppHost.Sdk` reference, you might still be using .NET Aspire 8. To upgrade to 9.0, follow [the upgrade guide](../get-started/upgrade-to-aspire-9.md).
+
+## 🚧 App host project file changes
+
+The .NET Aspire app host project file no longer requires the `IsAspireHost` property. This property was moved to the `Aspire.AppHost.Sdk` SDK, therefore, you can remove it from your project file. For more information, see [dotnet/aspire issue #8144](https://github.com/dotnet/aspire/pull/8144).
+
+<!--
+## 📈 Dashboard usage telemetry
+
+.NET Aspire collects usage telemetry from the dashboard by default. This telemetry helps us understand how you use the dashboard and what features are most important to you. We use this information to prioritize our work and improve the dashboard experience. You can opt out of this telemetry by setting the `DOTNET_DASHBOARD_ENABLE_TELEMETRY` environment variable to `false`. For more information, see [.NET Aspire dashboard usage telemetry](../fundamentals/dashboard/usage-telemetry.md).
+-->
+
+## 🚫 Disable dashboard resource graphs
```

Comment:
I feel like this is something strange to promote some prominently. Maybe we should have an errata section in the bottom for this?

Reply:
+1. This was just added for folks who are hosting the dashboard, have resources, but don't have relationships, so the graph doesn't add value.

The average user won't use this and won't care that it's there.